charts/s3gw/values.yaml

```diff
@@ -61,15 +63,20 @@ useExistingSecret: false
 # 'defaultUserCredentialsSecret' the name of the secret containing
 # the S3 Access Key and the S3 Secret Key for the default user.
 defaultUserCredentialsSecret: "s3gw-creds"
-# 'accessKey' is the S3 Access Key; the value is used when useExistingSecret: false.
-# Set this as the empty string to make the Chart to compute a random alphanumeric value.
+# 'accessKey' is the S3 Access Key; the value is used when
+# `useExistingSecret: false`.
+# Set this as the empty string to make the Chart to compute a random
+# alphanumeric value.
 accessKey: "test"
-# 'secretKey' is the S3 Secret Key; the value is used when useExistingSecret: false
-# Set this as the empty string to make the Chart to compute a random alphanumeric value.
+# 'secretKey' is the S3 Secret Key; the value is used when
+# `useExistingSecret: false`.
+# Set this as the empty string to make the Chart to compute a random
+# alphanumeric value.
 secretKey: "test"
 # 'publicDomain' is the public domain of S3GW used by the Ingress
-publicDomain: "be.127.0.0.1.omg.howdoi.website"
-# 'privateDomain' is the private domain of S3GW used inside the Kubernetes cluster
+publicDomain: "example.com"
```

Comment:
I'd rather prefer the user to be forced to put an explicit value.
I think it is better, for the public domain variables, to require the user to make the choice.

We can implement the following logic inside a `chart-validations.yaml` file:

```
{{- if (empty .Values.publicDomain) }}
{{- fail "publicDomain cannot be empty" }}
{{- end }}
```